spring-data-jdbc-plus-support/src/main/java/com/navercorp/spring/data/jdbc/plus/support/convert/SqlGenerator.java

```diff
@@ -106,12 +116,10 @@
  * VERIFIED: 91ddda0c3f97858e75cbf44ee23054559dd0daeb
  */
 @SuppressWarnings("checkstyle:linelength")
-class SqlGenerator {
+public class SqlGenerator {
```

Comment:
There were some changes in several commits.
However, I made changes following the last file.